### DISCUSSION TITLE
gm_construct_flatgrass_v6

### DISCUSSION THREAD

**Discussion by Xandertron** [TAServers/issues#20] [Addon Suggestions]
https://steamcommunity.com/sharedfiles/filedetails/?id=104468359
only fuckin map with actual decent train tracks

**Comment by ELPatitoLoco** [comment]
A variant of this map was posted some time ago would be cool if you voted it.

**Reply by ELPatitoLoco** [reply]
Well, since the map I'm talking about is more like a modification of this and its name its not listed this is an actual better post for it, and btw would be cool if we got this map added

**Comment by Derpius** [comment]
![image](https://github.com/TAServers/issues/assets/49565664/bb70f0c3-5c1a-4fa8-8fd7-1842136ef60e)

**Reply by Xandertron** [reply]
our default map is 114mb, normal bigcity and other maps are sub 40mb

**Reply by Derpius** [reply]
so what you're saying is our default map is half the size of this, and others are even smaller?

**Reply by Xandertron** [reply]
yes

**Comment by ELPatitoLoco** [comment]
The map also things you can control with wiremod its really cool, also its a big map.

**Comment by Derpius** [comment]
Added now at popular request...

We'll be migrating to nanos and/or undertaking work to improve cold join times in gmod, so not too fussed about the file size
